src/app/market/sell/add-item/add-item.component.ts

```diff
@@ -372,7 +372,10 @@ export class AddItemComponent implements OnInit, OnDestroy {
           this.log.d(listing);
           this.backToSell();
         },
-        (error) => this.snackbar.open(error),
+        (error) => {
+          this.isInProcess = false;
```

Comment:
rm

Reply:
complete() isn't called after error, it does not behave like finally() for some odd reason. I wasn't aware.